dockerfiles/Dockerfile.centos7

```diff
@@ -12,18 +12,27 @@ RUN sed -i -e "s/^mirrorlist=http:\/\/mirrorlist.centos.org/#mirrorlist=http:\/\
     cyrus-sasl-lib cyrus-sasl-devel openssl openss-libs openssl-devel \
     postgresql-libs postgresql-devel postgresql-server postgresql libyaml-devel && \
     yum install -y epel-release && \
-    yum install -y cmake3
+    yum install -y cmake3 && \
+    yum install -y centos-release-scl && \
```

Comment:
Do we need to add all this to the actual package build container definitions for CentOS 7?
`./packaging/distros/centos/Dockerfile`